www/src/index.tsx

```diff
@@ -1,12 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
```

Comment:
IMO we shouldn't alias this. I like just `BrowserRouter`. I don't see the point of the alias when there is no conflict here.

Reply:
Sounds good :+1: